devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -67,9 +68,14 @@ def installed(){
 def parse(String description) {
 	log.debug "description(): $description"
 	def map = zigbee.getEvent(description)
+	def manufacturer = getDataValue("manufacturer")
 	if(!map){
 		if (description?.startsWith('zone status')) {
-			map = parseIasMessage(description)
+			if (manufacturer == "eWeLink"){
```

Comment:
I think we may be able to avoid this manufacturer specific check which would be preferable. Does the eWeLink report the ZoneStatus message with the Restore Notify (bit 5) set? If so, and the other devices that use this DTH have Restore Notify cleared then we can just look at that bit to know whether we need to report motion stopped automatically or if the device will do it.

Reply:
@Duncankn can you let us know whether the PS601/z1 reports ZoneStatus with Restore Notify cleared?

Reply:
@tpmanley our device doesn't support Zone status with restore notify

Reply:
@tpmanley  I don't know well about the " Restore Notify", for the eWeLink device, if there is no motion during 1 min, the device will report 0x0000, and if there is detected motion, device will report 0x0001